README.md

```diff
@@ -27,25 +27,30 @@ jobs:
       - put: eks-kubectl
         params:
           kubectl: get nodes
+      - put: eks-kubectl
+        params:
+          kubectl: rollout restart deployment/my-deployment
+          namespace: my-other-namespace
 ```
 
 ## Why?
 Because we needed a simple script to authenticate to AWS and do a simple command for us.
 We also needed different kubectl versions from the ones maintained, so, here we are
 
 ## Resource definition - Source
-| Parameter Name        | Description                    |
-|-----------------------|--------------------------------|
-| aws_access_key_id     | AWS KEY (mandatory)            |
-| aws_secret_access_key | AWS Secret (mandatory)         |
-| eks_cluster_name      | EKS cluster name (mandatory)   |
-| eks_region            | EKS cluster region (mandatory) |
-| namespace             | Kubernetes namespace (option)  |
+| Parameter Name        | Description                     |
+|-----------------------|---------------------------------|
+| aws_access_key_id     | AWS KEY (mandatory)             |
+| aws_secret_access_key | AWS Secret (mandatory)          |
+| eks_cluster_name      | EKS cluster name (mandatory)    |
+| eks_region            | EKS cluster region (mandatory)  |
+| namespace             | Kubernetes namespace (optional) |
 
 ## PUT - Params
-| Parameter Name | Description             |
-|----------------|-------------------------|
-| kubectl        | Command you wish to run |
+| Parameter Name | Description                                                 |
+|----------------|-------------------------------------------------------------|
+| kubectl        | Command you wish to run                                     |
```

Comment:
**Suggestion:**

Maybe we could use something like this in the description, to make it easier for people that have limited knowledge regarding Kubernetes
`[Kubectl command](https://kubernetes.io/docs/reference/kubectl/) you wish to run`

Reply:
I think it is useful to add what @pedrodsrodrigues suggested.